flink-runtime/src/main/java/org/apache/flink/runtime/memory/MemoryManager.java

```diff
@@ -604,7 +582,7 @@ public int computeNumberOfPages(double fraction) {
 		}
 
 		//noinspection NumericCastThatLosesPrecision
-		return (int) (budgetByType.maxTotalNumberOfPages() * fraction / numberOfSlots);
```

Comment:
looooks like it is in fact used? 😕

Reply:
as it was supposed to be the last commit :), at this point `numberOfSlots` can be only 1, so it is needed anymore